packages/react-components/source/scss/library/components/_drawer.scss

```diff
@@ -0,0 +1,46 @@
+.rc-drawer-container {
+  min-width: fit-content;
+
+  .rc-drawer-header {
+    align-items: center;
+    border-top: $puppet-common-border;
+    display: flex;
+    justify-content: space-between;
+    min-height: 40px;
+    min-width: 100%;
+    overflow: hidden;
+  }
+
+  .rc-drawer-header-closed {
+    border-bottom: $puppet-common-border;
+  }
+
+  .rc-drawer-header-content-container {
+    border-right: $puppet-common-border;
+    flex-grow: 1;
+    height: 100%;
+    min-height: 40px;
+    overflow: hidden;
+    padding-right: 80px;
+  }
+
+  .rc-drawer-header-content-container-open {
+    border-bottom: $puppet-common-border;
+  }
+
+  .rc-drawer-toggle-container {
+    display: flex;
+    min-width: 136px;
+    padding: 0 5px;
+
+    .rc-drawer-toggle-button {
+      margin: auto;
+    }
+  }
+
+  .rc-drawer-body {
+    border-bottom: $puppet-common-border;
+    padding: 20px 50px;
+    transition: max-height 2s ease;
```

Comment:
This transition is not working as you are not changing the height/max-height when the drawer is opened. To support this you will probably need to always render the drawer body instead of conditionally rendering it based on `isOpen`. You can then add an `open` class to the drawer body div and set an initial max-height, overflow hidden and any other css you might need to hide the content while its closed.

Reply:
Talked to John Murray about this, he says that cause the sidepanel in relay doesn't currently have an animation we should be okay to proceed without one. There is also no requirement for it with estate reporting. however we both think it is a nice to have so i will update the current ticket and add a new one to the backlog to add animation down the line